### PR TITLE
Add flow sftp step

### PR DESCRIPTION
**Pull request by jaypha**

### CODE REVIEW

File: classes/local/step/sftp_trait.php [hunk 1]
```diff
@@ -0,0 +1,362 @@
...
+                return true;
+            }
+
+            // Upload to remote.
+            $this->upload($sftp, $sourcepath, $targetpath);
+        } finally {
+            $sftp->disconnect();
+        }
+
+        return true;
```
Comment (keevan) [inline classes/local/step/sftp_trait.php:262]: I believe this needs to return `$input|false` as it should work for both a connector and a flow.

**Review by keevan — COMMENTED** [review]
There also needs to be a version bump. 

The rest looks good. Thanks Jason.

**Comment by jaypha** [comment]
Bumped and rebased

**Review by keevan — APPROVED** [review]
Looks good to me

**Pull request merged**